OpenEdXMobile/src/main/java/org/edx/mobile/module/analytics/SegmentAnalytics.java

```diff
@@ -1097,4 +1098,20 @@ public void trackProfileScreenEvent(@NonNull String eventName, @NonNull String b
         aEvent.properties.putValue(Keys.NAME, biValue);
         trackSegmentEvent(eventName, aEvent.properties);
     }
+
+    @Override
+    public void trackVideoDownloadQualityClicked(String eventName, String biValue) {
+        final SegmentEvent aEvent = new SegmentEvent();
+        aEvent.properties.putValue(Keys.NAME, biValue);
+        trackSegmentEvent(eventName, aEvent.properties);
+    }
+
+    @Override
+    public void trackVideoDownloadQualityChanged(@NonNull VideoQuality selectedVideoQuality, @NonNull VideoQuality oldVideoQuality) {
+        final SegmentEvent aEvent = new SegmentEvent();
+        aEvent.properties.putValue(Keys.NAME, Analytics.Values.VIDEO_DOWNLOAD_QUALITY_CHANGED);
+        aEvent.data.putValue(Keys.VALUE, selectedVideoQuality.name().toLowerCase().replaceAll("option_", ""));
```

Comment:
Can we replace  `name().toLowerCase().replaceAll("option_", "")` with `getValue()`
PS: do change it on other places

Reply:
added.